package.json

```diff
@@ -15,14 +15,17 @@
   "homepage": "https://netboot.xyz",
   "dependencies": {
     "ejs": "3.1.10",
-    "express": "4.19.2",
+    "express": "4.21.0",
     "http": "0.0.0",
+    "http-proxy-agent": "7.0.2",
+    "https-proxy-agent": "7.0.5",
     "isbinaryfile": "5.0.2",
     "js-yaml": "4.1.0",
     "node-downloader-helper": "2.1.9",
-    "readdirp": "3.6.0",
+    "proxy-from-env": "1.1.0",
+    "readdirp": "4.0.1",
```

Comment:
Starting from `v4.0.0` there are breaking changes introduced in `readdirp`. All were addressed in the code itself.